app/assets/images/payment_icons/touchngo.svg

```diff
@@ -0,0 +1 @@
esEXG5br/ZW0/mdf//T66FPlyzdcOR0xLlruTVOXbkW2O3jpJVvfv/nd90+SfplT+aF1uavF+jlYbFEbdaGnW9+ujHlpXnJ+waAiIiIiIjKVbhNrDY2+hnrscDUDHdlWMRke/WVVOA5LO7+wNSMDtYnNOEhi+1dYKKOibNpHgigvTY22u4TURNNeMg12FphfqCOibOZr587fEiF6+QBQBMectp6zCdhcX/zY21sdKw2Ntpi14Qqsvf9E1WJdOky5KLSXZgFNzcIAQG2NVev30srmjrO1QT/64GJdPmyw7qq6tKk/mE/T3eLLVMjm9TfVMPdzSKkAICo1k2st1stfLh58CbrupZ1a1rMPNubdbFr+5kLtred/vXuuuPiL6/ce9RiCYu5VnVrHYJV4DJq1cYFE3/dMtN0XN/Pu8D8fLHB6Dtj/Y4vX13+e/lstMlRXT4EUFh26PFXRtbAnvErdgdNmJPX9N2E7OSTZ58F4JFXVBLS9eOk35NPni3fRaVNvToHhOvXIlOjfeDtVRsXDF/6W3lPEiIiIiIichJwGNJSvrc37jsvaXpgaoYQmJoheA6LU1T2o+yaGuqYOJstU/V7d9cE4PBGw0QZFvGw9ZghLSVBEx5i009Dlxh/jyY8RAZg+fgaQGBqxvPWY2fotLoAAAa7SURBVJ7D4hpbj2nCQ2RtbLTN19Tv3e2tCQ/RwbbB6A+q9h315gOGtJQEQ1rKDk14yDVdYrzD6fAAoI2NtgmP1DFxFjd8sLN2QBsb/Zbpc11ifIQmPMTmyTPdvXJ698O1Lg/iamgYilaUvq0DdifD8+03LOoKJr9X/rnn0CGo8Ynt5AbFfS3gv7F0x1b9jp0oWbPOpubfmPZEd4v3wKfPPjLOXt2MJ7t/5KFSlve3eK1H+xlNawfYNPdNm/By1LtRXUaIglAeRKSdvdT54D9XIvQGo/Beny7D3ZUK6/cYACC0Xq1zR6fE3BcU4LPb3nmT7s2ClsMqCEHZLi8A8PT9zf+Svxqv/uCpHkPNC8zfyD4eblnvRnUZWfjFO/d2bdqgfLvoZrX9s09NG9G4RZ2AzSAiIiIiIocEWZZRFgI4IgWmZlRurvoNKttVxOIpqDnrJpua8JBkAOXbSVoxonR3FG846NHhk7A4SNW+41l755z8LvIAGAA42sJRF5ia4WU+oAkPOQygpZ3arzyHxY1Tx8Tl6vfuFvR7U1oWzo//GUBTq7pjgakZ95kPaMJDtgGIhBOew+LuqajZKf13aJq1svlz6/5sf9SYORUAcDUiEvK1sk2DPNzhu2wxlK1bQZYk5I18HfqNZfmgIMB7/ldw6961/HXy3hgF47HjEGrUgOrBHvB86QUIXmrIej20zw6C8XD5Si8L/js2u4m1almEfZVxOU/nvSz18FMABAB446HwhY5qV+49+ugFbf69APBYy0Zb7run5hlHtQfOXQpeknK4/57Mi10EAXL7oHu2D45ovapt/dpZ85L3DSjSGzwA4MHmwTva1Kt9wvzagmK9+MvBjMcPnLvc4qI2v2GTWv6H3+3dJdG85vcjpyOST54L/0eb18RolBSiIOi+HdrHppdRYvK+57acyOp4RqNtXmQwevh4uOW+0rntgifaNPvdx9O9wiWAE37e/PqF3PxGkiSLKoXC8PWQqFEV1RMRERER3U0qE3AAAHwSFitV7TvaTBP/t7Sx0Y8a0lIsmv5Zs7eLSFnvjfZ2yiukDIto4zsvKR0OOAtbKpAXmJrhYz2oCQ/JAtDATn2l2PvZdYnxLQvnx9v0H7Dj28DUjJedFZHrq1LAAUAMalA+C0PSaqHtPxBSZmkPXyEwEL7LFkHRqKH1S1rInzQFxStWOTx/owEHERERERHRjahwiYq53OFDDJrwkE+c1VWWLjG+sSY8JN9ZuOFIYGpGB2VYxEBndWYKfBIWu1cUbgDlW7gKAI5XVGflC3vhBgAEpmYElS19qWpoUuhoe9iy5qgWW2E64KCTJN3tpKyzkKXS5qCiry+84z8BPEt3KZY1GmgHDkHJps2AbD/7lLKzKww3iIiIiIiIbrVKz+CwogPwPgCts0IrbgCGArDYJcAZRzf6JtrY6LcNaSkfwX5gc9JzWNzj6pg4m2agzuj37vbKHT5kNYAHHZRc9BwWF6WOidvn4LwF/d7dYu7wIWMATEDpEhp7Mj2HxfVWx8Q5naGhjY1+y5CWYi90+tNzWNzz6pg4my1n6b/J3gwO1YPdUWPG/wEQkPNIL8g6y11VPUfEwCN6ICCUvm2KlixH4ZyvLGqUHcLgNXkCxMBAyMXFMJ7+GyWbt6Dk59WQCyrO7DiDg4iIiIiIbiVTwLEKgMXOH3eQpYGpGYOdFRHdzewFHLcbAw4iIiIiIrqVRAAITM2w2Q72TsFwg4iIiIiIiIicKV/SEZiaISjDImZXVHyLzXa2NIWIyqhUzipuLYUiQ/DyqvamxERERERERI4Isp0mgprwkJEAngcQYHPy5roKYHlgasZcZ4X0/+3bMSvEcQDGcfenLM5lICy8CN1+q5u8gRsMivJ6lEFRt1oMUsdrYXB3GU9dEcdoUb/ljx59PvPzCr71wJfnw+PL18HdXmn3WxbaOyet/vlRaQcAAFCXbwMHkOV9NF6Z9PZPZ/cPf343qzbWr5Yvzg7mt7fGpS0AAEBdBA74Jz6m0+rl+qYzexxulrY/pVpbfVrs7t42mktvpS0AAECdBA4AAAAgXjUHAAAAEE7gAAAAAOIJHAAAAEA8gQMAAACIJ3AAAAAA8QQOAAAAIJ7AAQAAAMQTOAAAAIB4AgcAAAAQT+AAAAAA4gkcAAAAQDyBAwAAAIgncAAAAADxBA4AAAAgnsABAAAAxBM4AAAAgHgCBwAAABBP4AAAAADiCRwAAABAPIEDAAAAiCdwAAAAAPEEDgAAACCewAEAAADEEzgAAACAeAIHAAAAEE/gAAAAAOIJHAAAAEC8TwWyRAxRSHvjAAAAAElFTkSuQmCC"/></svg>
```

Comment:
This one also

![image](https://user-images.githubusercontent.com/658169/104206259-4f779380-53fd-11eb-868d-a61606ae07df.png)